client/Ratings.jsx

```diff
@@ -7,69 +7,108 @@ class Ratings extends React.Component {
   constructor(props) {
     super(props);
   }
+
+  getAverage(array, key) {
+    let total = 0;
+    const arrLength = array.length
+    array.map(obj => {
+      total += obj.ratings[key]
+    })
+    return total/arrLength;
+  }
+
   render() {
-    return (
-      <div className={styles.ratings}>
-        <div className={styles.ratingsLeft}>
-          <div className={styles.checkIn}>
-            Check-in
-            <div className={styles.pbCheckin}>
-              <progress max="5" value="4.5"></progress>
+    if (!this.props.searchActive) { // if search is not active, display the ratings
+      return (
+        <div className={styles.ratings}>
+          <div className={styles.ratingsLeft}>
+            <div className={styles.checkIn}>
```

Comment:
given this div has similar logic that is repeated a few times below for each rating, a possibility is to consolidate and refactor to have one centralized function or component to handle the logic of assigning classNames, getting the average, assigning the Description, etc. This could potentially help with debugging and readbility.

Reply:
That is definitely a goal during this week if I have time! Thank you for the feedback!